tests/testing_common.py

```diff
@@ -38,13 +39,22 @@
 
 
 CONFIG_CLASSES = (
+    AdaMixConfig,
     IA3Config,
     LoraConfig,
     PrefixTuningConfig,
     PromptEncoderConfig,
     PromptTuningConfig,
 )
 CONFIG_TESTING_KWARGS = (
+    {
+        "target_modules": None,
+        "adapter_dim": 16,
+        "num_expert": 4,
+        "sharing_down": False,
+        "sharing_up": True,
+        "return_two_views": True,
+    },
```

Comment:
Can we get a 2nd config with slightly different params, e.g.:

```
    {
        "target_modules": None,
        "adapter_dim": 8,
        "num_expert": 3,
        "sharing_down": True,
        "sharing_up": False,
        "return_two_views": False,
    },
```

That way, we get better test coverage.